network.sh

```diff
@@ -259,6 +259,12 @@ function add_dnsmasq_multi_entry() {
     #     None
     for i in ${2//${VIPS_SEPARATOR}/ }; do
         if [ "${1}" = "apivip" ] ; then
+            if [[ "$IP_STACK" == "v4" || "$IP_STACK" == "v4v6" ]]; then
+               echo "server=/${CLUSTER_DOMAIN}/$(nth_ip $EXTERNAL_SUBNET_V4 1)" | sudo tee -a "${PATH_CONF_DNSMASQ}"
+            fi
+            if [[ "$IP_STACK" == "v6" ]]; then
+               echo "server=/${CLUSTER_DOMAIN}/$(nth_ip $EXTERNAL_SUBNET_V6 1)" | sudo tee -a "${PATH_CONF_DNSMASQ}"
+            fi
```

Comment:
This change does not seem required to me (and maybe also potentially harmful for other dev-scripts flows, non-agent related). If that is required by haproxy, I think it'd be a simpler approach to just enrich the host DNS (defined by `$PATH_CONF_DNSMASQ`) with just the nodes name (in addition to api and api-int)

Reply:
Yes, enriching the host DNS in $PATH_CONF_DNSMASQ would also work. Will update.